apps/drec-ui/env.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+REQUIRED_VARIABLES=(
```

Comment:
If you are using `prod-start.env` and replacing ui env variables on the start time, then you should also use our `dev-env.sh` file.
And implement the flow with adding env variable into `Window` inside `index.html.template`.
Also make sure you are using env variables like `window.config.BACKEND_URL` not the process.env
Then you can remove this file.